packages/rocketchat-lib/server/startup/settings.js

```diff
@@ -180,7 +180,7 @@ RocketChat.settings.addGroup('Accounts', function() {
 	});
 
 	this.section('Accounts_Default_User_Preferences', function() {
-		this.add('Accounts_Default_User_Preferences_enableAutoAway', false, {
+		this.add('Accounts_Default_User_Preferences_enableAutoAway', true, {
```

Comment:
the auto away default value as `false` was added my mistake by https://github.com/RocketChat/Rocket.Chat/pull/7285 , so I'm reverting back to `true`